actor/src/main/scala/com/evolutiongaming/akkaeffect/Act.scala

```diff
@@ -72,12 +80,20 @@ private[akkaeffect] object Act {
         val value = {
           class Main
           new Main with Act[F] {
+
+            @volatile var stopped = false
+
+            def postStop(): Unit =
+              stopped = true
+
             def apply[A](f: => A) =
               for {
                 adapter <- Sync[F].delay(threadLocal.get())
                 result <- {
                   if (adapter.contains(self: Adapter[F])) {
                     Sync[F].delay(f)
+                  } else if (stopped) {
+                    stoppedError.raiseError[F, A]
```

Comment:
why this check is not the first one? why we even try to get `adapter` if actor could be `stopped`?

Reply:
Mostly to avoid checking is the actor is stopped if the code executed from within the actor, afaik its most of the use-cases

Reply:
is it useful to get the `threadLocal` if actor is stopped?

like, if actor is alive, then we will get `threadLocal` and if `f` is async, we will check for `stopped`, right?

but when actor is stopped, then we will get `threadLocal` and if `f` is async, we will check the `stopped`...

does it mean that we mostly have only "sync" calls for `f` and rarely "async" ones? and we allow "sync" calls even when actor is `stopped`?

Reply:
> does it mean that we mostly have only "sync" calls for f and rarely "async" ones?

at least its my assumption. I cannot prove it and do not expect that anybody can prove opposite 

> we allow "sync" calls even when actor is stopped?

its not possible to have `sync` call while actor is stopped because the only place from where one can do `sync` call is actor's `receive` function. After stopping
```
adapter.contains(self: Adapter[F])
``` 
will always be `false`

Reply:
oh, the `stopped` is flag only for "async" flow... if so, then I'd move the check inside "async" handling block to show the "binding" or document the behaviour.

Reply:
Thanks, I've added explanation to the var:
```
            // flag used to prevent async (i.e. `tell` to self ) operations after actor is stopped
            // because they will be lost as actor is already stopped.
            @volatile var stopped = false
```